### DISCUSSION TITLE
Best method for multiple people training the same model

### DISCUSSION THREAD

**Discussion by clee758** [talmolab/sleap#1389] [General]
Hi,

I'm planning on training a model with someone else and was wondering what the best method to do so is. I was thinking of either labeling half of the videos and having them label the other half, then combining videos into one project to train one model. My other idea was to label half of the frames in all of the videos and export a package so they could label the other half of the frames on their computer. Or is there another better way for us to both label frames? What is the best method?

**Comment by talmo** [comment]
Hi @clee758,

Yep, I think both of those strategies should work well. Separating it by either video or suggestions works and will ensure that you don't have merge conflicts when you combine them again.

The exported package approach sometimes has issues depending on the video source and a few other factors, so I'd do a test run of exporting -> labeling on another machine -> saving -> merging, just to make sure nothing weird happens with your project (like duplicate videos being created or something).

Let us know if you have any questions!

Cheers,

Talmo
